src/frontend/components/SubscribeScheduledVideoEmailForm/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Notification,
+  StatusType,
+  TextInput,
+} from 'grommet';
+import React, { Fragment, useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import { createLiveRegistration } from '../../data/sideEffects/createLiveRegistration';
```

Comment:
Can you use absolute imports in place of ascending relative ones in this file please ?